pandas/tests/io/test_sql.py

```diff
@@ -2310,7 +2310,6 @@ def test_schema(self):
         cur = self.conn.cursor()
         cur.execute(create_sql)
 
-    @tm.capture_stdout
```

Comment:
are these actually outputing things?

Reply:
@jreback in some of the tests there is now some output displayed when using -s option of `pytest`. 

with `pytest` output is captured by default. so disabling output is not necessary.

https://docs.pytest.org/en/latest/capture.html#default-stdout-stderr-stdin-capturing-behaviour

"During test execution any output sent to stdout and stderr is captured. If a test or a setup method fails its according captured output will usually be shown along with the failure traceback."

in my view this should be the desired behaviour.

`tm.capture_stdout` disables output entirely and the output is not available with the -s option of `pytest`

if this behaviour is really considered necessary, it could be done by adding a `capsys.readouterr()` to clear the buffer at the end of the test, applied as a decorator as it is now, or maybe as a fixture with the clearing the buffer in the test teardown and then applying using `@pytest.mark.usefixtures` decorator

Reply:
ok that's fine then.